payments-core/src/main/java/com/stripe/android/auth/PaymentBrowserAuthContract.kt

```diff
@@ -28,14 +27,7 @@ internal class PaymentBrowserAuthContract :
         val shouldUseBrowser =
             input.hasDefaultReturnUrl(defaultReturnUrl) || input.isInstantApp
 
-        val statusBarColor = when (context) {
-            is Activity -> context.window?.statusBarColor
-            else -> null
-        }
```

Comment:
The general approach: No longer use a default value for `statusBarColor` in the args class, which will force us to actually set it before `createIntent()` is called.